### ISSUE TITLE
Compiled library will not work in C#

### ISSUE THREAD

**Issue by wrharper-AASP** [tdlib/td#2426]
I am getting a:
'Could not load file or assembly 'Telegram.Td, Version=0.0.0.0, Culture=neutral, PublicKeyToken=null'. The system cannot find the file specified.'
error for some reason it can't recognize the dll even though it loads into the project and is compiled with 0 errors.

I have tried:
1. using old ssl dlls since it tries using 1_1 by default and vcpkg uses 3.
2. upgrading the project to 2022 and manually configuring/building via C++ Visual Studio 2022.
3. built all 4 versions via the cmake instructions and none of them work.
4. using any CPU as per past posts about this same issue.

This needs to be possible on the .NET 6 framework in visual studio 2022.

**Comment by levlam** [comment]
You need to use https://tdlib.github.io/td/build.html?language=C%23 to get fully working build instructions. README can only mention general steps. You can also combine the generated build instructions with description from the README to get the example working out of the box. For example, ues commit `1b1ae50e1a69f7c659bd7d731e80b358d21c86ad` of `vcpkg` to build OpenSSL 1.1.1.

The C# example uses C++/CLI to interact with native code, which should be supported by .NET 6 Framework on Windows.

**Comment by wrharper-AASP** [comment]
is it possible to get the readme updated in: https://github.com/tdlib/td/tree/master/example/csharp to these instructions?

**Comment by levlam** [comment]
Just use `git checkout 1b1ae50e1a69f7c659bd7d731e80b358d21c86ad` before calling `.\vcpkg.exe install ...` to install expected dependencies.

**Comment by wrharper-AASP** [comment]
Right, this worked, and I am good, thanks for that, but the readme is misleading.
If the readme gets updated, people wouldn't have to keep making posts about it.
